targets/targets.json

```diff
@@ -4177,17 +4178,14 @@
             }
         },
         "overrides": { "lse_available": 0 },
-        "macros_add": ["HSE_VALUE=26000000", "VECT_TAB_OFFSET=0x08010000"],
-        "post_binary_hook": {
-            "function": "MTSCode.combine_bins_mts_dragonfly",
-            "toolchains": ["GCC_ARM", "ARM_STD", "ARM_MICRO", "IAR"]
-        },
-        "device_has_add": ["MPU"],
+        "macros_add": ["HSE_VALUE=26000000"],
+        "device_has_add": ["MPU", "FLASH"],
```

Comment:
How come we dont need this post binary hook anymore?

Reply:
That was Multitech's bootloader. We don't need that bootloader to boot. In addition, we can't fit their bootloader, our bootloader and cloud client on this board. Even without their bootloader, only develop and release profile work.

Reply:
I see, we can remove them then (this only removes reference in targets.json, but bootloaders are still in tools or ?)